src/MADE.UI.Controls.ChipBox/Chip.cs

```diff
@@ -0,0 +1,108 @@
+// MADE Apps licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace MADE.UI.Controls
+{
+    using System.Windows.Input;
+    using MADE.UI.Extensions;
+    using Windows.UI.Xaml;
+    using Windows.UI.Xaml.Controls;
+
+    /// <summary>
+    /// Defines a control for displaying a value as a chip with remove capabilities.
+    /// </summary>
+    [TemplatePart(Name = ChipContentPart, Type = typeof(ContentPresenter))]
+    [TemplatePart(Name = ChipRemoveButtonPart, Type = typeof(Button))]
+    public sealed partial class Chip : ContentControl, IChip
```

Comment:
Do you need to provide an automation peer for the `Chip`?

Reply:
Sorted